node/send_http_request/index.js

```diff
@@ -0,0 +1,37 @@
+const axios = require("axios");
+module.exports = async (req, res) => {
+  let url, method, body, headers;
+
+  try {
+    const payload = JSON.parse(req.payload);
+    url = payload.url;
+    method = payload.method || undefined;
+    headers = payload.headers || undefined;
+    body = payload.body || undefined;
+  } catch (err) {
+  
+    res.json({ success: false, message: "URL could not be reached." });
```

Comment:
Let's update the error message here. Could be something along likes `Payload does not have a proper structure.`.

Reply:
Done with changes